modules/jaeger-config-storage.adoc

```diff
@@ -189,18 +189,18 @@ spec:
 [id="jaeger-config-external-es_{context}"]
 == Connecting to an existing Elasticsearch instance
 
-Jaeger also allows you to use an existing (self-provisioned) Elasticsearch cluster for storage. You do this by specifying the URL of the existing cluster as the `spec:storage:options:es:server-urls` value in your configuration.
+Jaeger also allows you to use an already existing Elasticsearch cluster for storage (that is, an instance that was not auto-provisioned by the Jaeger Operator calling the Red Hat Elasticsearch Operator). You do this by specifying the URL of the existing cluster as the `spec:storage:options:es:server-urls` value in your configuration.
```

Comment:
I'll leave this final decision to you, but I try to avoid using parenthesis. It's not a big deal to use it, and it reads fine here, but the parenthesis can imply a lack of importance. 

Consider revising it to something like, `...an existing Elasticsearch cluster for storage, for an example: an instance that was not auto-provisioned by the Jaeger Operator calling the Red Hat Elasticsearch Operator. `

Reply:
> ! allow verb
Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...".

Also, +1 to avoiding parentheses:

> Avoid using parentheses in general text because text in parentheses is decreased in importance.